pkg/cmd/step/e2e/step_e2e_gc_test.go

```diff
@@ -46,10 +46,10 @@ func TestShouldDeleteOlderThanDuration(t *testing.T) {
 	o := e2e.StepE2EGCOptions{}
 	o.Duration = 2
 	cluster := getCluster("168", "jenkins-gkebdd", "159")
-	cluster.ResourceLabels["create-time"] = time.Now().Format("Mon-Jan-2-2006-15-04-05")
+	cluster.ResourceLabels["create-time"] = time.Now().UTC().Format("Mon-Jan-2-2006-15-04-05")
```

Comment:
@daveconde `ShouldDeleteOlderThanDuration` in _step_e2e_gc_ does take the time in UTC time (https://github.com/jenkins-x/jx/blob/c28614230d4e12925df0a7a0377c0ff479b6d4fb/pkg/cmd/step/e2e/step_e2e_gc.go#L144), so the tests need to do the same. For me this test was failing since I am two hours ahead of UTC.